tests/commonlib/io_utils.py

```diff
@@ -100,14 +100,79 @@ def exec_command(container_name: str, command: str, param_value: str, resource:
 
         current_resource = Path(resource)
         if not current_resource.is_file():
-            raise Exception(f"File {resource} does not exist or mount missing.")
+            raise Exception(
+                f"File {resource} does not exist or mount missing.")
 
         if command == 'chmod':
             os.chmod(path=resource, mode=int(param_value))
         elif command == 'chown':
             uid_gid = param_value.split(':')
             if len(uid_gid) != 2:
-                raise Exception("User and group parameter shall be separated by ':' ")
+                raise Exception(
+                    "User and group parameter shall be separated by ':' ")
             shutil.chown(path=resource, user=uid_gid[0], group=uid_gid[1])
         else:
-            raise Exception(f"Command '{command}' still not implemented in test framework")
+            raise Exception(
+                f"Command '{command}' still not implemented in test framework")
+
+    @staticmethod
+    def edit_process_file(container_name: str, dictionary, resource: str):
+        if container_name == '':
+            raise Exception(f"Unknown {container_name} is sent")
+
+        current_resource = Path(resource)
+        if not current_resource.is_file():
+            raise Exception(
+                f"File {resource} does not exist or mount missing.")
+
+        with current_resource.open() as f:
+            r_file = yaml.safe_load(f)
+
+        command = r_file["spec"]["containers"][0]["command"]
+        set_dict = dictionary.get("set", {})
+        unset_list = dictionary.get("unset", [])
+
+        for skey, svalue in set_dict.items():
```

Comment:
Not very easy to understand from the first glimpse, an explanation or even simplifying code will be great.